easybuild/easyconfigs/s/SLATEC/SLATEC-4.1-GCC-13.2.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'SLATEC'
+version = '4.1'
+
+homepage = 'https://www.netlib.org/slatec/'
+description = """SLATEC Common Mathematical Library, a comprehensive software library containing
+    over 1400 general purpose mathematical and statistical routines written in Fortran 77."""
+
+toolchain = {'name': 'GCC', 'version': '13.2.0'}
+
+sources = [{
+    'source_urls': ['https://www.netlib.org/%(namelower)s'],
+    'filename': '%(namelower)s_src.tgz',
+    'extract_cmd': 'tar xfvz %s',
+}, {
+    'source_urls': ['http://www.netlib.org/%(namelower)s'],
+    'filename': '%(namelower)s4linux.tgz',
+    'extract_cmd': 'tar xfvz %s -C src',
+}]
+patches = ['%(name)s-%(version)s-slatec4linux-fPIC.patch']
+checksums = [
+    {'slatec_src.tgz': 'd03ab24415277fc423764798e3b86d2e9bd43f8409919837e14af506f736dd54'},
```

Comment:
Checksum doesn't seem stable for this source tarball...

With a fresh download, I get `4c8c02fee905325ee4906bf8f7ece5593d895da3e5f208322f8aacea6d0eb9dc`